code/core/system/proxy-fh16/include/CANMessageDataStore.h

```diff
@@ -47,11 +47,12 @@ namespace proxy {
  */
 class CanMessageDataStore : public automotive::odcantools::MessageToCANDataStore {
    public:
-    CanMessageDataStore(std::shared_ptr< automotive::odcantools::CANDevice > canDevice);
+    CanMessageDataStore(std::shared_ptr< automotive::odcantools::CANDevice > canDevice, bool readOnly = false);
```

Comment:
Sorry, missed this before. I think the default should be 'true', we should always have the safest possible mode by default.